### DISCUSSION TITLE
@swc/helper error when using the latest version of headless ui

### DISCUSSION THREAD

**Discussion by defiboyszn** [tailwindlabs/headlessui#3243] [Help]
I'm getting this error when using headless ui latest versions from v2+
error - ./node_modules/@react-aria/interactions/dist/usePress.mjs:3:0
Module not found: Can't resolve '@swc/helpers/_/_class_private_field_get'

Import trace for requested module:
./node_modules/@react-aria/interactions/dist/import.mjs
./node_modules/@headlessui/react/dist/components/button/button.js
./node_modules/@headlessui/react/dist/headlessui.esm.js
./src/layout/index.tsx
./src/pages/_app.tsx

https://nextjs.org/docs/messages/module-not-found

**Comment by prathamesh-dukare** [comment]
Same, Im also getting the same!

**Comment by andresespinosapc** [comment]
This also happens to me, have you been able to solve it?

**Comment by major-next** [comment]
Is this resolved now?

**Comment by phips28** [comment]
we have the same issue.

**Comment by stevenpslade** [comment]
This is likely due to a conflicting package dependency on a different version of `@swc/helper`. In my case, I am using Next.js at version `12.2.5` which has a dependency on version `0.4` (`@headlessui` depends on `^0.5.0`).

I was able to resolve this thanks to [this comment](https://github.com/vercel/next.js/issues/48593#issuecomment-1681930868) by creating resolution of next's `@swc/helper` to a higher version:
```
// package.json
"resolutions": {
    "next/@swc/helpers": "0.4.36"
  }
```

Updating to a higher version of `next` that depends on a different version of `@swc/helper` would also work and is my long term plan.

**Comment by lpite** [comment]
try to use "@headlessui/react": "1.7.19" its works with nextjs@12
